oap-server/server-starter/src/main/resources/ui-initialized-templates/so11y_satellite/so11y-root.json

```diff
@@ -0,0 +1,236 @@
+[
+  {
+    "id":"Self-Observability-Satellite-Root",
+    "configuration":{
+      "children":[
+        {
+          "x":8,
+          "y":0,
+          "w":8,
+          "h":14,
+          "i":"1",
+          "type":"Widget",
+          "widget":{
+            "title":"Connection Count"
+          },
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "standard":{
+            "unit":"Count"
```

Comment:
The `standard` has been abandoned, if not `Card` chart, the unit should type on `title` directly. Like `Connection Count (Count)`.

Reply:
@wallezhang Please try to set up a new one? This may be caused by old imported template. 
If not, we need a UI fix.

Reply:
just remove it manually, re-import it and export it could fix

Reply:
I just make the standard as empty.

> re-import it and export it could fix

I have tried, but the `unit` field still exists.